.github/workflows/email-check.yaml

```diff
@@ -0,0 +1,36 @@
+name: "Check for private emails used in PRs"
+on: pull_request_target
+permissions:
+  pull-requests: write
```

Comment:
Nit: We normally try and add additional permissions in the job and set `permissions.contents` to read here so that if additional jobs end up getting added to the workflow, they don't have unnecessary permissions.

Reply:
Sure, can do that